src/app/zeko/circuits/ase.ml

```diff
@@ -111,6 +111,8 @@ module With_length = struct
       (({ source; target } : Stmt.var), verifier)
 
     let make = Made_2.make
+
+    let get_iterations = get_iterations
```

Comment:
You should hard-code the values in your sequencer instead of expecting the circuits to export these values.

The circuits in the future will have a heavily simplified API as described in #241

Reply:
Also I would expect that after #241 the code would move somewhere else, not disappear.

Reply:
Will put them in zeko constants.